blockly/src/server/Server.java

```diff
@@ -1415,4 +1422,63 @@ public void push() {
       }
     }
   }
+
+  /**
+   * Attempts to pull an entity in front of the hero.
+   *
+   * <p>If there is a pushable entity in the tile in front of the hero, it checks if the tile behind
+   * the player is accessible. If accessible, the hero and the entity are moved simultaneously in
+   * the opposite direction the hero is facing.
+   *
+   * <p>The pulled entity temporarily loses its blocking component while moving and regains it
+   * after.
+   */
+  public void pull() {
```

Comment:
@AMatutat Das ist eine 1:1 Kopie von `push()`, oder? ... Nur korrigiert um die Bewegungsrichtung. 

Wäre es nicht sinnvoller, eine gemeinsame Methode zu haben, die als Parameter noch die Richtung bekommt und die Arbeit macht und die von `push` und `pull` aufgerufen wird?

Reply:
yep,

bin auch schon dabei.

Mann muss eigentlich nur unterscheiden
- welche richtung
- welche tlles to check